notebooks/official/pipelines/google_cloud_pipeline_components_TPU_model_train_upload_deploy.ipynb

```diff
@@ -61,7 +63,11 @@
       "source": [
         "## Overview\n",
         "\n",
-        "This notebook shows how to use the components defined in [`google_cloud_pipeline_components`](https://github.com/kubeflow/pipelines/tree/master/components/google-cloud) in conjunction with an experimental `run_as_aiplatform_custom_job` method, to build a [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines) workflow that trains a [custom model](https://cloud.google.com/vertex-ai/docs/training/containers-overview) using TPUs, uploads the model as a `Model` resource, creates an `Endpoint` resource, and deploys the `Model` resource to the `Endpoint` resource.\n",
+        "This notebook shows how to use the components defined in [`google_cloud_pipeline_components`](https://github.com/kubeflow/pipelines/tree/master/components/google-cloud) SDK in conjunction with an experimental `run_as_aiplatform_custom_job` method, to build a [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines) workflow that:\n",
```

Comment:
- change to:  "This notebook shows how to use the components defined in the [`google_cloud_pipeline_components`](https://github.com/kubeflow/pipelines/tree/master/components/google-cloud) SDK in conjunction with the  [Vertex AI Pipelines'](https://cloud.google.com/vertex-ai/docs/pipelines) experimental method, `run_as_aiplatform_custom_job`, to build a pipeline workflow that:"

Reply:
Removing the usage of the experimental method as it's no longer needed in the latest commits. 
Changed this sentence to "This notebook shows how to use the components defined in the [google_cloud_pipeline_components](https://github.com/kubeflow/pipelines/tree/master/components/google-cloud) SDK in conjunction with [Vertex AI Pipelines](https://cloud.google.com/vertex-ai/docs/pipelines) to build a pipeline that:"